README.md

```diff
@@ -3,14 +3,118 @@
 [![Build Status](https://dev.azure.com/cnlabs/porter/_apis/build/status/deislabs.porter?branchName=master)](https://dev.azure.com/cnlabs/porter/_build/latest?definitionId=6?branchName=master)
 
 
-# Porter is a Cloud Installer
+# Porter is a tool for building cloud installers
 
 Porter gives you building blocks to create a cloud installer for your application, handling all the
 necessary infrastructure and configuration setup. It is a declarative authoring experience that lets you
 focus on what you know best: your application.
 
 Learn more at [porter.sh](https://porter.sh)
 
----
+*Want to work on Porter with us? See our [Contributing Guide](CONTRIBUTING.md)*
 
-Want to work on Porter with us? See our [Contributing Guide](CONTRIBUTING.md)
\ No newline at end of file
+## Getting Started
```

Comment:
Just a heads up that the porter docs all live at https://porter.sh/docs/ which is why the readme points people there. We are slowly building up docs over there, and some already exist (such as the install guide).

Would you please move the getting started to a new file located at /docs/content/quickstart.md and then link to it from the README here?

The readme can have links to useful stuff for new contributors, but I'd like to avoid repeating content so that we aren't dual editing. 😀 

Thank you for adding this! 🎉